.github/workflows/build-and-publish-images.yaml

```diff
@@ -54,7 +73,9 @@ jobs:
         if: steps.publish.outcome == 'skipped'
         uses: ./.github/workflows/template-publish-image
         with:
-          image: docker.io/kubeflowkatib/${{ inputs.component-name }}
+          image: |
+            ghcr.io/kubeflow/katib/${{ inputs.component-name }}
+            docker.io/kubeflow/katib/${{ inputs.component-name }}
```

Comment:
ditto

Reply:
i'll revert it to prevent docker failure. thanks for bringing this up.

Reply:
done.